Source/WebCore/html/shadow/SliderThumbElement.cpp

```diff
@@ -75,7 +75,8 @@ inline static Decimal sliderPosition(HTMLInputElement& element)
 inline static bool hasVerticalAppearance(HTMLInputElement& input)
 {
     ASSERT(input.renderer());
-    return input.renderer()->style().effectiveAppearance() == SliderVerticalPart;
+    // Target vertical writing modes regardless of the appearance, or appearance: slider-vertical;
```

Comment:
This comment says exactly what the code below it does, just in English rather than C++. In WebKit, we try to leave out those comments. We could either not comment here at all, or write a comment that says *why*, since that kind of comment would add something that the code below does not say.

Reply:
will remove.